src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -112,8 +111,16 @@ protected override void ContextUnloaded(ExecutionContext context)
         }
 
         public static ApplicationEngine Create(TriggerType trigger, IVerifiable container, StoreView snapshot, long gas, bool testMode = false)
```

Comment:
The parameters of the `Create()` method must be consistent with the constructor.

Reply:
Done, please take a look